CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+# This source file is part of the Swift.org open source project
+#
+# Copyright (c) 2014 - 2023 Apple Inc. and the Swift project authors
+# Licensed under Apache License v2.0 with Runtime Library Exception
+#
+# See http://swift.org/LICENSE.txt for license information
+# See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+
+cmake_minimum_required(VERSION 3.28)
```

Comment:
Fairly sure we're 3.24 still everywhere else

Reply:
Oh, yeah, it's kind of all over the place, and doesn't really matter.  3.24 works for me.

Reply:
Let's make sure it's consistent with whatever the Swift repo does. Fair?

Reply:
Yeah, I'll double check.  Like I said, we're not doing anything here that requires a significantly advanced version.